internal/jobs/twitter.go

```diff
@@ -113,19 +113,19 @@ func (ts *TwitterScraper) getAuthenticatedScraper(j types.Job, baseDir string, j
 	var scraper *twitter.Scraper
 
 	switch jobType {
-	case TwitterCredentialScraperType:
+	case string(teetypes.TwitterCredentialJob):
```

Comment:
OK, one more. `jobType` should be a `JobType` (with all the upstream changes that entails).

Now that I think about it, why do we need the `jobType` parameter when we already have the job? (not necessarily to fix now, just to ponder for when we do cleanup)